### DISCUSSION TITLE
Can't see cover images in Github Pages

### DISCUSSION THREAD

**Discussion by bimgeek** [adityatelange/hugo-PaperMod#569] [Q&A] [answered]
Hi guys,

I am quite new around here. I created my first website using Hugo and PaperMod theme and hosted it in a github repo. You can view my website from [here](https://bimgeek.github.io). The problem that I am facing is that I created a cover image for each single page. I can view them just fine in my local. But I can't view them online. 
![image](https://user-images.githubusercontent.com/51519350/135125141-b615a7f0-d944-4a8e-9cec-7a3945e52bd6.png)

I inspected it and found out after I delete lazy loading, I can see the images. Any idea why I can't view cover images in list pages?

Thanks.

![image](https://user-images.githubusercontent.com/51519350/135125400-6e3b5444-6666-40a6-a2b0-71085870abca.png)

![image](https://user-images.githubusercontent.com/51519350/135125367-eaa32dfd-4ed2-4eed-98e5-ab9de87f746d.png)

**Comment by danielfdickinson** [comment]
You appear not to have uploaded the generated responsive images.

**Reply by bimgeek** [reply]
How can I do that?
My github repo is [here](https://github.com/bimgeek/bimgeek-website)

**Reply by bimgeek** [reply]
seems like they exist in the repo.

**Comment by adityatelange** [comment, marked as answer]
Hi @bimgeek,

I see that you have your baseURL set with `http` instead of `https`. Please change it to `https` and try.

https://github.com/bimgeek/bimgeek-website/blob/b2fbef917e9b9d8d1f683f5c82a3ccc15d823208/config.toml#L1

**Reply by bimgeek** [reply]
Yesss, this solves the problem. Thanks @adityatelange
